Makefile

```diff
@@ -1,13 +1,13 @@
-all             :; DAPP_BUILD_OPTIMIZE=1 DAPP_BUILD_OPTIMIZE_RUNS=200 dapp --use solc:0.6.12 build
-clean           :; dapp clean
+all             :; FOUNDRY_OPTIMIZER=true FOUNDRY_OPTIMIZER_RUNS=200 forge build --use solc:0.6.12
+clean           :; forge clean
 # usage:        make test match=Burn
 test            :; ./test.sh ${match}
 deploy          :; echo "use deploy-goerli or deploy-mainnet"
 deploy-goerli   :; make && ./scripts/deploy-goerli.sh
 deploy-mainnet  :; make && ./scripts/deploy-mainnet.sh
 flatten         :;
-	hevm flatten --source-file "src/UniswapV2Callee.sol" > out/UniswapV2CalleeDai.sol
-	hevm flatten --source-file "src/UniswapV2LpTokenCallee.sol" > out/UniswapV2LpTokenCalleeDai.sol
-	hevm flatten --source-file "src/UniswapV3Callee.sol" > out/UniswapV3Callee.sol
-	hevm flatten --source-file "src/WstETHCurveUniv3Callee.sol" > out/WstETHCurveUniv3Callee.sol
-	hevm flatten --source-file "src/CurveLpTokenUniv3Callee.sol" > out/CurveLpTokenUniv3Callee.sol
+	forge flatten "src/UniswapV2Callee.sol" > out/UniswapV2CalleeDai.sol
+	forge flatten "src/UniswapV2LpTokenCallee.sol" > out/UniswapV2LpTokenCalleeDai.sol
+	forge flatten "src/UniswapV3Callee.sol" > out/UniswapV3Callee.sol
+	forge flatten "src/WstETHCurveUniv3Callee.sol" > out/WstETHCurveUniv3Callee.sol
+	forge flatten "src/CurveLpTokenUniv3Callee.sol" > out/CurveLpTokenUniv3Callee.sol
```

Comment:
does it works the same? I think there were some open issues with `forge flatten`

Reply:
tested now, and with the path `out/flatten.sol` doesn't seem to work

Reply:
The diff seems ok - https://www.diffchecker.com/ilFDi7qj/
Was also able to verify the contract using forge flatten.
So I think we are good.

Reply:
see: https://github.com/foundry-rs/foundry/issues/4034

Reply:
not sure if we're safe here but worth double checking

Reply:
also, with forge to specify output path it seems to be preferred the use of the option `--output` (see [here](https://book.getfoundry.sh/reference/forge/forge-flatten?highlight=forge%20flatten#examples))

Reply:
Addressed in #37